docs/index.rst

```diff
@@ -0,0 +1,44 @@
+.. GenomicsDB documentation master file, created by
+   sphinx-quickstart on Wed Jun 10 21:36:23 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
```

Comment:
Could we add a title block to the top of the index? Currently the landing page shows up as `<no title> -- GenomicsDB` in the browser tab.

Reply:
Done - in my defense, I have too many tabs open to ever see the title for my browser tabs 😆